priv/riak_shell/riak_shell_regression1.log

```diff
@@ -24,6 +24,20 @@
 |GeoCheckin|
 +----------+
 "}}.
+{{command, "EXPLAIN SELECT * FROM GeoCheckin WHERE myfamily = 'family1' AND myseries = 'series1' AND time >= 2 AND time <= 7000000 AND weather='fair';\n"}, {result, "+-----+-------------------------------------+------------+-------------------------------------+-----------+----------+
+|Subqu|        Range Scan Start Key         |Is Start Inc|         Range Scan End Key          |Is End Incl|  Filter  |
++-----+-------------------------------------+------------+-------------------------------------+-----------+----------+
+|  1  |myfamily = 'family1', myseries = 'ser|   false    |myfamily = 'family1', myseries = 'ser|   false   |weather = |
```

Comment:
The columns are truncated, so the "Filter" value isn't visible. This is also the case for the "Range Scan Start Key" column and other column displays.